kivymd/uix/button/button.py

```diff
@@ -937,6 +943,7 @@ class ButtonContentsIconText(ButtonIconMixin):
     Contents for BaseButton consisting of a BoxLayout
     with an icon and a label.
     """
+
     pass
```

Comment:
It is not necessary to specify the `pass` instruction

Reply:
Also removed.